RecoPPS/RPixEfficiencyTools/plugins/EfficiencyTool_2018DQMWorker.cc

```diff
@@ -1053,10 +1061,14 @@ void EfficiencyTool_2018DQMWorker::analyze(const edm::Event &iEvent, const edm::
   Handle<reco::ForwardProtonCollection> multiRP_protons;
   iEvent.getByToken(multiRP_protonsToken_, multiRP_protons);
 
-  edm::ESHandle<LHCInfo> lhcInfo;
-  iSetup.get<LHCInfoRcd>().get("", lhcInfo);
-  const LHCInfo *pLhcInfo = lhcInfo.product();
-  double xangle = pLhcInfo->crossingAngle();
+  // edm::ESHandle<LHCInfo> lhcInfo;
+  // iSetup.get<LHCInfoRcd>().get("", lhcInfo);
+  // const LHCInfo *pLhcInfo = lhcInfo.product();
+  auto const& dataLHCInfo2 = iSetup.getData(lhcInfoToken_);
+  const LHCInfo *pInfo2 = &dataLHCInfo2; //TODO: change code to use reference isntead of pointer
```

Comment:
Why do you want to use the reference instead of the pointer?

Reply:
It was a suggestion made by Chrys. It's just a better programming practice. 
Firstly, we have one line of code less.
Secondly, if possible it's better to use a reference than a pointer - it provides a guarantee that an object cannot be changed (for a pointer - only a pointer is constant and a thoughtless programmer can change data)

Reply:
Awesome, looks nice!